cmake/traccc-compiler-options-hip.cmake

```diff
@@ -0,0 +1,12 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021-2024 CERN for the benefit of the ACTS project
```

Comment:
In the R&D projects I prefer to date the files with a starting year that they were created in. So in this case I would write:

```suggestion
# (c) 2024 CERN for the benefit of the ACTS project
```

But this is purely a subjective thing.